pkg/e2etemplates/aws_operator_chart_values_test.go

```diff
@@ -0,0 +1,297 @@
+package e2etemplates
+
+import "testing"
+
```

Comment:
This thing here is used in all the tests. It creates fully filled `AWSOperatorChart` and takes simple function which allows modifying it. `modifyFunc` is used just to remove fields to test validation and optional fields.